tests/actions/test_core_actions.py

```diff
@@ -12,16 +14,18 @@
     ValidationResultIdentifier,
 )
 from great_expectations.validation_operators import (
+    MicrosoftTeamsNotificationAction,
     OpsgenieAlertAction,
     PagerdutyAlertAction,
     SlackNotificationAction,
     StoreValidationResultAction,
 )
 
-try:
-    from unittest import mock
-except ImportError:
-    from unittest import mock
+
+class MockTeamsResponse:
+    def __init__(self, status_code):
+        self.status_code = status_code
+        self.text = "test_text"
```

Comment:
Mock the returned response object for MS Teams

Reply:
A similar pattern may also be useful in testing other validation operators like Slack and Pagerduty